lib/Storage/TimeSeriesMongoDbResults.php

```diff
@@ -58,13 +58,14 @@ public function __construct($cursor = null)
     /** @inheritdoc */
     public function getArray()
     {
+        $this->object->setTypeMap(['root' => 'array']);
         return $this->object->toArray();
     }
 
     /** @inheritDoc */
     public function getIdFromRow($row)
     {
-        return $row['id'];
+        return (string)$row['id'];
```

Comment:
this (string) is not required. Id comes as string as well after setting the typeMap to array